pymc3/tests/test_distributions.py

```diff
@@ -284,6 +284,10 @@ def mvt_logpdf(value, nu, Sigma, mu=0):
     return log_pdf
 
 
+def AR1_logpdf(value, k, tau_e):
+    return (sp.norm(loc=0,scale=1/np.sqrt(tau_e)).logpdf(value[0]) +
```

Comment:
But yeah, this test is a bit odd as it basically repcliates the likelihood. `statsmodels` is an idea but we don't want that as a dependency, but could be optional. @junpenglao?